### ISSUE TITLE
Compile error with base-4.6

### ISSUE THREAD

**Issue by hvr** [fizruk/http-api-data#42]
I'd suggest to simply set a lower bound `base >= 4.7` if you don't want to support base < 4.7

```
Configuring http-api-data-0.3.3...
Preprocessing library http-api-data-0.3.3...

src/Web/Internal/FormUrlEncoded.hs:50:18:
    Could not find module `Data.Proxy'
    It is a member of the hidden package `tagged-0.8.5'.
    Perhaps you need to add `tagged' to the build-depends in your .cabal file.
    It is a member of the hidden package `tagged-0.8.4'.
    Perhaps you need to add `tagged' to the build-depends in your .cabal file.
    Use -v to see a list of the files searched for.
```

**Comment by hvr** [comment]
Fyi, I've revised the meta-data for the affected releases, e.g. https://hackage.haskell.org/package/http-api-data-0.3.3/revisions/

So there's no immediate need to act. Just make sure the next release (whenever that happens) includes that fix as well.

**Comment by phadej** [comment]
For the record: 0.3 version uses closed type families, so GHC >= 7.8 is hard requirement.

**Comment by fizruk** [comment]
Closed by #44.
